js/core/css/var.css

```diff
@@ -0,0 +1,21 @@
+var:hover {
+  border-bottom: 1px dotted;
+  cursor: pointer;
+}
+
+var.respec-active {
+  background: var(--respec-background-color);
+  color: var(--respec-color);
+}
+
+var.respec-bug {
```

Comment:
I'm thinking maybe we shouldn't do this one. There might be legit cases for having a single `var`. 
At least, we should be really sure before we show anything red... or people are going to get upset :)

Reply:
will remove that then.